zeg/tests/test_imagesets.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Zegami Ltd
+
+"""Imageset tests."""
+
+from . import HTTPBaseTestCase
+from .. import imagesets
+
+
+class ImagesetTestCase(HTTPBaseTestCase):
+    def test__update_to_url_imageset(self):
```

Comment:
I wouldn't include leading underscores to make two underscores in a test method name, ends up confusing.